### PR TITLE
awsume 4.0.0 (new formula)

### PR DESCRIPTION
**Pull request by GGeethanjali**
Added new formula - awsume 4.0.0

### CODE REVIEW

**Comment by chenrui333** [comment]
@GGeethanjali Can you update the commit message format as `formular version (new formula)`? Thanks!

File: Formula/awsume.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Awsume < Formula
+  include Language::Python::Virtualenv
+
+  desc "Utility for easily assuming AWS IAM roles from the command-line"
+  homepage "https://www.trek10.com/blog/awsume-aws-assume-made-awesome"
+
+  url "https://github.com/trek10inc/awsume/archive/4.0.0.tar.gz"
+  sha256 "8cc7d466beef1c32b3397ed62cba22f7a309ab1650317cef6873cf019be7d25e"
+  head "https://github.com/trek10inc/awsume.git", :branch => "master"
```
Comment (bfontaine) [inline Formula/awsume.rb]: Isn’t `:branch => "master"` already the default?

Comment (GGeethanjali) [reply Formula/awsume.rb]: Got it, will remove the redundant info

**Comment by chenrui333** [comment]
13:50:12 ==> /usr/local/Cellar/awsume/4.0.0/bin/awsume -v
13:50:12 4.0.0
13:50:12 Error: awsume: failed
13:50:12 An exception occurred within a child process:
13:50:12   ArgumentError: wrong number of arguments (given 0, expected 1)

**Comment by chenrui333** [comment]
Thanks @GGeethanjali for your contribution to Homebrew! :tada: 🥇 

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Comment by fxcoudert** [comment]
```
depends_on "openssl"
```

No, please don't 😢 
We are actively getting rid of OpenSSL 1.0: https://github.com/Homebrew/homebrew-core/issues/46454 and it should definitely not be introduced in new formulas

**Pull request closed without merging**